xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -2635,12 +2635,14 @@ bool COutput::GLInit()
   }
 #endif
 
+#ifdef HAS_GL
```

Comment:
ok, could you please put a comment in the code with reference to this. when I get time I'll check if GL_TEXTURE_RECTANGLE_ARB is still required for GL. I think that can be dropped but not completely sure.